pages/web-literacy-skills.jsx

```diff
@@ -0,0 +1,2 @@
+//TODO clean up indirection
+module.exports = require('./web-literacy/Skills.jsx');
```

Comment:
do we need this file? If this is a new file, we don't, whatever needs to load this page can just directly `require('pages/web-literacy/Skills.jsx');`